packages/styles/scss/patterns/blocks/content-block-simple/_content-block-simple.scss

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+
+@mixin content-block-simple {
+  .#{$prefix}--content-block-simple {
+    &__content {
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    .#{$prefix}--content-item {
+      margin-bottom: $carbon--spacing-07;
```

Comment:
margin bottom here I think might be more determined based on the CTA. `32` is correct unless the CTA is a `.bx--card` in which then it needs to be `48px`. The `48px` I think is only for `@md` and higher.

I wonder if that's necessary, or if we could see if design is willing to having only one spacing between either CTA. @wonilsuhibm @oliviaflory @larahanlon2